pom.xml

```diff
@@ -1498,14 +1499,40 @@
               <artifactId>maven-javadoc-plugin</artifactId>
               <version>2.9.1</version>
               <configuration>
-                <stylesheetfile>cdap-distributions/src/javadoc/stylesheet.css</stylesheetfile>
-                <links>
-                  <link>http://download.oracle.com/javase/6/docs/api/</link>
-                </links>
-                <doctitle>${project.name} ${project.version}</doctitle>
                 <bottom>
                   <![CDATA[Copyright &#169; {currentYear} <a href="http://cask.co" target="_blank">Cask Data, Inc.</a> Licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0" target="_blank">Apache License, Version 2.0</a>.]]>
                 </bottom>
+                <!--  Turn debug to true to retain the files used by the plugin with javadoc (javadoc.sh, options) -->
+                <debug>false</debug>
+                <doctitle>${project.name} ${project.version}</doctitle>
+                <!--  To use excludePackageNames, sourcepath needs to be specified -->
+                <excludePackageNames>co.cask.cdap.internal:co.cask.cdap.template.etl.common.guice</excludePackageNames>
+                <groups>
+                  <group>
+                    <title>Core Packages</title>
+                    <packages>co.cask.cdap.api*</packages>
+                  </group>
+                  <group>
+                    <title>Application Template Packages</title>
+                    <packages>co.cask.cdap.template.*</packages>
+                  </group>
+                </groups>
+                <links>
+                  <link>http://download.oracle.com/javase/${project.java.version}/docs/api/</link>
+                </links>
+                <maven.javadoc.failOnError>false</maven.javadoc.failOnError>
+                <quiet>true</quiet>
+                <!-- sourcepath needs to be specified to use excludePackageNames -->
+                <sourcepath>
+                  ${project.parent.basedir}/cdap-api/src/main/java:
```

Comment:
Well we have javadocs for some packages under cdap-api module isn't it? So I was wondering why is cdap-api/src/main/java being included here. Basically if it was working earlier, why was this not present already.